$NetBSD: patch-ab,v 1.9 2010/07/21 11:57:52 tron Exp $

Dragonfly PR pkg/43285
Same issue as PR pkg/43098 for devel/m4

--- lib/spawn.in.h.orig	2010-03-17 19:17:04.000000000 +0000
+++ lib/spawn.in.h
@@ -110,6 +110,7 @@ typedef struct
 
 
 /* Flags to be set in the `posix_spawnattr_t'.  */
+#if !@HAVE_POSIX_SPAWN@ || defined(__APPLE__)
 #if @REPLACE_POSIX_SPAWN@
 /* Use the values from the system, for better compatibility.  */
 /* But this implementation does not support AIX extensions.  */
@@ -122,6 +123,7 @@ typedef struct
 # define POSIX_SPAWN_SETSCHEDPARAM      0x10
 # define POSIX_SPAWN_SETSCHEDULER       0x20
 #endif
+#endif
 /* A GNU extension.  Use the next free bit position.  */
 #define POSIX_SPAWN_USEVFORK \
   ((POSIX_SPAWN_RESETIDS | (POSIX_SPAWN_RESETIDS - 1)                   \
